docs/example_csharp.md

```diff
@@ -206,23 +206,28 @@ With Tilt, we can skip all of these steps, and instead [live_update](live_update
 Here's our [new Tiltfile](https://github.com/tilt-dev/tilt-example-csharp/blob/master/3-live-update/Tiltfile) with the following new code:
 
 ```python
-docker_build(
+load('ext://restart_process', 'docker_build_with_restart')
+...
+docker_build_with_restart(
     'hello-tilt',
     'out',
+    entrypoint=['dotnet', 'hello-tilt.dll'],
     dockerfile='Dockerfile',
     live_update=[
         sync('out', '/app/out'),
     ],
-    entrypoint='find . | entr -r dotnet hello-tilt.dll',
 )
 ```
 
-We've added a `live_update` parameter to `docker_build()` with a `sync` step. This copies the build output from the `out` directory into container.
+The first thing to notice is the `live_update` parameter, which consists of one `sync` step.
+This copies the build output from the `out` directory into the container.
 
-We've also added a new parameter: `entrypoint='find . | entr -r dotnet hello-tilt.dll'`.
-
-`entr` is a tool that automatically restarts a shell command whenever the watched
-file changes. This command restarts our server every time the pod is updated.
+After syncing the files, we want to re-execute our updated binary. We accomplish this with the
+[`restart_process` extension](https://github.com/windmilleng/tilt-extensions/tree/master/restart_process)
```

Comment:
SGTM, I'd just add a short remark about what the `load(ext)` call is. E.g.

```suggestion
[`restart_process` extension](https://github.com/windmilleng/tilt-extensions/tree/master/restart_process), which we imported on the first line
```

Reply:
ooh good call, ty